src/currency/supplemental-override.js

```diff
@@ -8,16 +8,13 @@ define([
  * Return pattern with fraction digits overriden by supplemental currency data.
  */
 return function( currency, pattern, cldr ) {
-	var digits,
-		fraction = cldr.supplemental([ "currencyData/fractions", currency ]) ||
-			cldr.supplemental( "currencyData/fractions/DEFAULT" );
-
-	digits = +fraction._digits;
+	var fraction = "",
+		fractionData = cldr.supplemental([ "currencyData/fractions", currency ]) ||
+			cldr.supplemental( "currencyData/fractions/DEFAULT" ),
+		digits = +fractionData._digits;
```

Comment:
`digits` depends on `fractionData` to be set. Therefore, I prefer it on a new line. Can you revert that part, please? (this is, `var digits,` then `digits = `)